prow/config.yaml

```diff
@@ -19287,7 +19287,7 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
-- name: ci-kubernetes-kubemark-5-prow-canary
```

Comment:
Why are you removing the 5-node canary job?

Reply:
the job was never configured properly though